etcdctl/ctlv3/command/auth_command.go

```diff
@@ -40,6 +41,14 @@ func newAuthEnableCommand() *cobra.Command {
 	}
 }
 
+func newAuthenticateCommand() *cobra.Command {
```

Comment:
i do not think we should have this command for etcdctl. once we provide username and password, etcdctl should just call auth internally and send the request with the token to get the response we want.